threat-model/readme.md

```diff
@@ -159,6 +135,8 @@ An aggregator is an individual party which participates in an aggregation protoc
 
 If enough aggregators (beyond the proposal specific subset) collude e.g. by sharing unencrypted input shares), then none of the properties of the system hold. Such scenarios are outside the threat mode.
 
+However, we do assume that an attacker can always control at least one aggregator (i.e. there are no perfectly trusted aggregators.)
```

Comment:
This is intended to motivate the need for multiple coordinators with threshold keys (as otherwise, the coordinator is a single point party with the complete decryption key.) I agree this maybe not the right place to put this, and happy to think about this more, but I do think it's important that there isn't a single party which has the ability to fully decrypt the data.

For the cloud service provider, the claim here is that the TEE provides sufficient protection against intrusion that they are unable to access the data. (Different vendors have different opinions on if this is sufficient.)

For the software vendor, I'm not sure how this is distinct from the software used for an MPC implementation.

Reply:
For what it's worth, the recent release of privacy sandbox aggregation service uses two coordinators
https://github.com/privacysandbox/aggregation-service

Reply:
@palenica love to see it, thanks for pointing it out! I'm going to rework this whole section, and jump right to that design.

Reply:
I thought about this, but the coordinators purpose is not only to keep the key secret, but also to validate the attestation. If that validation also happens in a TEE, I think the client/user would also want to be able to validate an attestation that *that* TEE is configured correctly, and it ends up turtles all the way down.